src/main/java/org/mskcc/oncokb/config/DatabaseConfiguration.java

```diff
@@ -29,9 +31,13 @@
 @Profile("!" + JHipsterConstants.SPRING_PROFILE_CLOUD)
 @Import(value = MongoAutoConfiguration.class)
 @EnableMongoAuditing(auditorAwareRef = "springSecurityAuditorAware")
-public class DatabaseConfiguration {
+public class DatabaseConfiguration extends AbstractMongoConfiguration{
 
     private final Logger log = LoggerFactory.getLogger(DatabaseConfiguration.class);
+    @Value("${spring.data.mongodb.uri}")
```

Comment:
Do we have default uri and database in the configuration file? Otherwise, we need to specify here.

Reply:
We set different mongo uri in different config-XXX.yml files. When we use using development model, we use the application-dev.yml(mongodb://localhost:27017) to build war file. If we want to deploy it on heroku, then we'll use application-heroku.yml(${MONGODB_URI}?authMode=scram-sha1) and application-prod.yml to build the war file.